contribution.md

```diff
@@ -59,9 +66,13 @@ These files mean that the new version will not be built accidentally.
 
 Delete the corresponding `.exclude-$OS` files once the image can built using testing/production rpms.
 
+#### 3) enable publishing
+
+In order to have images available on the [quay.io](quay.io) the `.github/workflows/build-and-push.yml` configuration file need to be updated with the proper $OS and $VERSION.
```

Comment:
I would also add a link to quay.io repositories, where the containers are stored.
E.g. For quay.io/fedora leaves Fedora images, in quay.io/centos7 leaves CentOS 7 images and in quay.io/sclorg goes C8S, C9S images